src/js/utils/normalize.ts

```diff
@@ -0,0 +1,15 @@
+import { normalize } from '@sentry/utils';
+
+/**
+ * Converts any input into a valid record with string keys.
+ */
+export function convertToNormalizedObject(wat: unknown): Record<string, unknown> {
```

Comment:
L: Did you check if the JS SDK always has this? sounds like possible.

Reply:
I've checked, as there are a lot of these kinds of util methods, but nothing handy for this.